api/src/test/java/com/exelaration/abstractmemery/services/MemeServiceTest.java

```diff
@@ -31,7 +32,7 @@ public void init() {
   }
 
   @Test
-  public void save_WhenMemeSavedSuccessfully_ExpectMemeReturned() {
+  public void memeServicesave_WhenMemeSavedSuccessfully_ExpectMemeReturned() {
```

Comment:
do we need to specify `memeServiceSave` instead of just `save` in the test name when it is already in the `MemeServiceTest`?

Reply:
no

Reply:
I agree with Katheryn